backend/curriculum_tracking/tests/test_api_views.py

```diff
@@ -235,6 +246,47 @@ def test_set_project_link(self):
         self.assertEqual(project.link_submission, link_2)
 
 
+class TestLongRunningActorsCalledFromAgileCardAPIView(APITestCase, APITestCaseMixin):
+
+    LIST_URL_NAME = "agilecard-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
+    SUPPRESS_TEST_GET_LIST = True
+
+    def setUp(self):
+        JAVASCRIPT = "js"
+        self.super_user = factories.UserFactory(is_superuser=True)
+
+        self.content_item = factories.ContentItemFactory(
+            content_type=ContentItem.PROJECT,
+            project_submission_type=ContentItem.REPOSITORY,
+            flavours=[JAVASCRIPT],
+        )
+
+        self.card = factories.AgileCardFactory(
+            content_item=self.content_item,
+            flavours=[JAVASCRIPT],
+            status=AgileCard.READY,
+        )
+
+        self.assignee = SocialProfileFactory().user
+        self.card.assignees.set([self.assignee])
+        self.project = self.card.recruit_project
+        self.url = f"{self.get_list_url()}{self.card.id}/setup_project_repo/"
+
+    @mock.patch('curriculum_tracking.api_views.Response')
+    @mock.patch.object(long_running_request_actors.recruit_project_setup_repository, 'send')
+    def test_send_called_from_api_view_action_option(self, send, Response):
```

Comment:
No need to mock path response. And if there was a need to patch it then that string isn't very specific, the response is not defined in curriculum_tracking, we import it.

This works:

```
from dramatiq.message import Message

    @mock.patch.object(
        long_running_request_actors.recruit_project_setup_repository, "send"
    )
    def test_send_called_from_api_view_action_option(self, send):  

        self.login(self.super_user)
        send.return_value = Message(
            queue_name="queue", actor_name="actor", args=(), kwargs={}, options={}
        )   # send it meant to return a Message instance. So we just return one. Now it behaves in a predictable way. 
        self.client.post(path=self.url, data={"card_id": self.card.id})
        send.assert_called()
```